database/redis/metric.go

```diff
@@ -306,16 +306,17 @@ func (connector *DbConnector) RemoveMetricRetention(metric string) error {
 }
 
 // RemoveMetricValues remove metric timestamps values from 0 to given time
-func (connector *DbConnector) RemoveMetricValues(metric string, toTime int64) error {
+func (connector *DbConnector) RemoveMetricValues(metric string, toTime int64) (int64, error) {
```

Comment:
Maybe we should use named return parameter, so that it would be more obvious what does this int64 mean

Reply:
do u think is it ok use in one func in all pkg?